### DISCUSSION TITLE
Graphical representation of CTSEM

### DISCUSSION THREAD

**Discussion by rifatzahan** [cdriveraus/ctsem#50] [General]
Sorry, if I am asking a question whose solution already exists in the package documentation. I was wondering, if there exists a function in CTSEM package, that can draw the graphical representation of the model process? Like, tidySEM, graph_sem(). I wanted to draw the model path for a better visualization.

**Comment by cdriveraus** [comment]
ctModelLatex is the closest you'll get. The state space matrix representation is pretty compact and useful, worth getting used to I believe :)
